docs/For User/Imversed Governance/Submit a Proposal.md

```diff
@@ -0,0 +1,239 @@
+        "@type": "/cosmos.upgrade.v1beta1.SoftwareUpgradeProposal",
+        "title": "v3.5",
+        "description": "upgrade",
+        "plan": {
+          "name": "v3.5",
+          "time": "0001-01-01T00:00:00Z",
+          "height": "1322760",
+          "info": "",
+          "upgraded_client_state": null
+        }
+      },
+      "authority": "imv10d07y265gmmuvt4z0w9aw880jnsr700jemq2lk"
+    }
+  ],
+  "status": "PROPOSAL_STATUS_PASSED",
+  "final_tally_result": {
+    "yes_count": "500000000000000000000000",
+    "abstain_count": "0",
+    "no_count": "0",
+    "no_with_veto_count": "0"
+  },
+  "total_deposit": [
+    {
+    "denom": "aimv",
+    "amount": "500000000"
+    }
+  ],
+  "metadata": ""
+}
+```
+
+The deposit `denom` is `aimv` and `amount` is `20100000000000000000`. Therefore, a deposit of 20.1 IMV will be included with this proposal. At the time, the IMV mainnet had a 10 IMV minimum deposit, so this proposal was put directly into the voting period (and subsequently passed). The minimum deposit amount is currently 192 IMV. There is a minimum deposit required for a proposal to enter the voting period, and anyone may contribute to this deposit within a 3-day period. If the minimum deposit isn't reached before this time, the deposit amounts will be burned.
+
+### Sending the transaction that submits your governance proposal
+
+For information on how to use `imversed` binary to submit an on-chain proposal through the governance module, please refer to the [quickstart](https://docs.imversed.com/validators/quickstart/binary.html) documentation.
+
+#### CLI
+This is the command format for using imversed (the command-line interface) to submit your proposal on-chain:
+
+```linux
+imversed tx gov submit-proposal \
+  --title=<title> \
+  --description=<description> \
+  --type="Text" \
+  --deposit="1000000aimv" \
+  --from=<mykey> \
+  --chain-id=<chain_id>
+  --node <address>
+```
+
+> Use the `imversed tx gov --help` flag to get more info about the governance commands
+
+1. `imversed` is the command-line interface client that is used to send transactions and query Imversed
+2. `tx gov submit-proposal param-change` indicates that the transaction is submitting a parameter-change proposal
+3. `--from mykey` is the account key that pays the transaction fee and deposit amount
+4. `--gas 500000` is the maximum amount of gas permitted to be used to process the transaction
+   * the more content there is in the description of your proposal, the more gas your transaction will consume
+   * if this number isn't high enough and there isn't enough gas to process your transaction, the transaction will fail
+   * the transaction will only use the amount of gas needed to process the transaction
+5. `--gas-prices` is the flat-rate per unit of gas value for a validator to process your transaction
+6. `--chain-id imversed_555555-1` is Imversed Mainnet. For current and past chain-id's, please look at the Chain ID documentation.

```

Comment:
Change chain-id **imversed_555555-1** to i**mversed_5555555-1**

Reply:
updated